attributes/default.rb

```diff
@@ -1,5 +1,5 @@
-#ntp server name (single)
-default['w32time']['ntpserver'] = 'pool.ntp.org'
+#ntp server names
+default['w32time']['servers'] = Array['0.pool.ntp.org', '1.pool.ntp.org', '2.pool.ntp.org', '3.pool.ntp.org']
```

Comment:
OPT: you don't need to precise `Array` just `['string1', 'string2'] is enough`
OPT: use the string array short syntax: `%w(string1 string2)`